packages/form-js-viewer/src/render/components/util/sanitizerUtil.js

```diff
@@ -19,6 +20,15 @@ export function sanitizeDateTimePickerValue(options) {
   return value;
 }
 
+export function isDeepIncluded(value, array) {
```

Comment:
The name is not very descriptive, I think we can improve it

```suggestion
export function hasEqualValue(value, array) {
```